### DISCUSSION TITLE
1.7.1-beta3

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#875] [Обновления]
> [!WARNING]
> Бета версия может работать нестабильно. Обо всех проблемах, рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)

## Список изменений

- Обновлена логика запросов к VOT Worker для поддержки новых запросов из 1.7.1-beta2
- Обновлена логика нового плеера. Теперь, он не использует Tone.js и работает лучше, чем раньше

## What's Changed
* Changed audio context player Tone.js --> chaimu by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/874

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.7.1-beta2...1.7.1-beta3

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.7.1-beta3'>1.7.1-beta3</a>.</em>

**Comment by NullVerdict** [comment]
Если включить перевод и сменить видео, кнопка перестанет работать.

**Reply by ilyhalight** [reply]
вроде фикс
